custom_bundle_signing/Makefile

```diff
@@ -0,0 +1,26 @@
+build:
```

Comment:
The build task is what will get executed by the main Make script (see https://github.com/open-policy-agent/contrib/blob/master/Makefile#L7) so if you want to run any tests as part of the build they should be added here.

Reply:
Added a test to check the output of `opa sign`